third-party/linshare/src/test/java/org/apache/james/linshare/LinshareConfigurationTest.java

```diff
@@ -43,37 +45,80 @@ void shouldMatchBeanContract() {
     @Test
     void fromShouldThrowWhenUrlIsNull() {
         PropertiesConfiguration configuration = new PropertiesConfiguration();
-        configuration.addProperty(LinshareConfiguration.USER_NAME_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getUsername());
+        configuration.addProperty(LinshareConfiguration.UUID_PROPERTY, DEFAULT_UUID);
         configuration.addProperty(LinshareConfiguration.PASSWORD_PROPERTY, LinshareFixture.TECHNICAL_ACCOUNT.getPassword());
-        configuration.addProperty(LinshareConfiguration.URL_PROPERTY, null);
 
         assertThatThrownBy(() -> LinshareConfiguration.from(configuration)).isInstanceOf(MalformedURLException.class);
     }
 
     @Test
-    void fromShouldThrowWhenBasicAuthnIsNull() {
+    void fromShouldThrowWhenBasicAuthIsNull() {
         PropertiesConfiguration configuration = new PropertiesConfiguration();
-        configuration.addProperty(LinshareConfiguration.USER_NAME_PROPERTY, null);
-        configuration.addProperty(LinshareConfiguration.PASSWORD_PROPERTY, null);
```

Comment:
You didn't understand correctly previous comment from @chibenwa here.

He asked you to separate those in multiple tests, not removing those lines. You changed the meaning of the tests.
This is not when a property is null anymore, but when it's missing. 

Please add back those properties to null (same for other tests) and add extra tests for those same properties missing